app/src/main/java/emu/skyline/input/onscreen/OnScreenEditActivity.kt

```diff
@@ -54,35 +61,103 @@ class OnScreenEditActivity : AppCompatActivity() {
         toggleFabVisibility(false)
     }
 
+    private val colors = arrayOf(Color.GRAY, Color.argb(180, 0, 0, 0), Color.argb(180, 255, 255, 255), Color.argb(180, 230, 255, 0), Color.argb(180, 180, 0, 230), Color.argb(180, 255, 60, 40), Color.argb(180, 225, 15, 0), Color.argb(180, 10, 185, 230), Color.argb(180, 70, 85, 245), Color.argb(180, 30, 220, 0))
```

Comment:
Here you'd want to make an enumeration of all colors with a meaningful name (Black, White, Teal, Lime Green, etc...), then use it to compose the array of colors. Doing it like this makes it very hard to see what colors you have in there without having too look them up.

Reply:
Yep that's done